fairlearn/datasets/_fetch_adult.py

```diff
@@ -25,6 +25,9 @@ def fetch_adult(*, cache=True, data_home=None,
     Prediction task is to determine whether a person makes over $50,000 a
     year.
 
+    ..versionadded: 0.5.0
+    ..versionchanged: 0.7.1
```

Comment:
version changed should have a comment explaining what's changed.

Reply:
In this particular case nothing changed other than the documentation, so I think we can remove that line and keep the `versionadded` one only.

Reply:
Come to think of it, the same applies to the other two in this module.